.circleci/config.base.yml

```diff
@@ -899,6 +899,26 @@ jobs:
           name: Sanity check install
           command: |
             amplify version
+  deployment_verification_post_release:
+    <<: *linux-e2e-executor-large
+    steps:
+      - restore_cache:
+          key: amplify-cli-repo-{{ .Branch }}-{{ .Revision }}
+      - run:
+          name: Verify Release Deployment
+          command: |
+            version=$(cat .amplify-pkg-version)
+            yarn ts-node scripts/verify-deployment.ts -v $version
+  deployment_verification_rc_and_tagged:
+    <<: *linux-e2e-executor-large
+    steps:
+      - restore_cache:
+          key: amplify-cli-repo-{{ .Branch }}-{{ .Revision }}
+      - run:
+          name: Verify Tagged and RC Deployment
+          command: |
+            version=$(cat .amplify-pkg-version)
+            yarn ts-node scripts/verify-deployment.ts --version $version --prerelease true
```

Comment:
nit: `yargs` support so called boolean flags. I.e. the sole presence of flag indicates it's `true`.